examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+meanio-controller-zwk1b                      10.244.0.13         meanio                  shaiweinstein/k8s:v1                kubernetes-minion-yxea.c.mean-io.internal/104.155.36.70   app=meanio,name=meanio,uses=meanio-mongodb                                  Running             23 hours
+meanio-mongodb-controller-nyxxv              10.244.0.11         meanio-mongodb          mongo:2.6.8                         kubernetes-minion-yxea.c.mean-io.internal/104.155.36.70   app=mongodb,name=meanio-mongodb                                             Running             23 hours
+
+```
+
+You will see a mongodb server pod, and meanio pod.
+
+### Step Four: Create the meanio service.
+
+Just like the others, you want a service to group your meanio pods (in case of running more than one relica)..
+The service is described in the file `examples/meanio/meanio-service.json`:
+
+```js
+{
+  "id": "meanio",
+  "kind": "Service",
+  "apiVersion": "v1beta1",
+  "port": 80,
+  "containerPort": "meanio-server",
+  "selector": {
+    "name": "meanio"
+  },
+  "labels": {
+    "name": "meanio"
+  },
+  "createExternalLoadBalancer": true
+}
+```
+
+If running a single node local setup, or single VM, you don't need `createExternalLoadBalancer`.
+
+```shell
+$ kubectl create -f examples/meanio/meanio-service.json
+meanio
+
+
+$ kubectl get services
+NAME                    LABELS                                    SELECTOR                     IP                  PORT
+meanio                  name=meanio                                                     name=meanio                  10.0.29.25          80
+meanio-mongodb          name=meanio-mongodb                                             name=meanio-mongodb          10.0.222.168        27017
+```
+
+### A few Google Container Engine specifics for playing around with the services.
+
+In GCE, `cluster/kubectl.sh` automatically creates forwarding rule for services with `createExternalLoadBalancer`.
+
+```shell
+$ gcloud compute forwarding-rules list
+NAME                  REGION      IP_ADDRESS     IP_PROTOCOL TARGET
+kubernetes-meanio europe-west1 130.211.xxx.xxx TCP         europe-west1/targetPools/kubernetes-meanio
+```
+
+You can grab the external IP of the load balancer associated with that rule and visit `http://130.211.xxx.xxx`.
+
+In GCE, you also may need to open the firewall for port 80 using the [console][cloud-console] or the `gcloud` tool. The following command will allow traffic from any source to instances tagged `kubernetes-minion`:
+
+```shell
+$ gcloud compute firewall-rules create --allow=tcp:80 --target-tags=kubernetes-minion kubernetes-minion-80
+```
+
+For GCE details about limiting traffic to specific sources, see the [GCE firewall documentation][gce-firewall-docs].
+
+[cloud-console]: https://console.developer.google.com
+[gce-firewall-docs]: https://cloud.google.com/compute/docs/networking#firewalls
```

Comment:
Would you be willing to add a tiny section at the bottom that links to the guestbook example for how to do cleanup, in general? I don't want you to have to write your own cleanup section, as it would be very similar to the guestbook guide, but it would be nice to direct people over there so they can at least pattern-match the `kubectl` calls and cleanup their example.